### PR TITLE
Node structure sample

### PR DESCRIPTION
**Pull request by asalauyeu-hubspot**
Added sample app for nodejs. Adding general readme with link to node folder. Current logic looks like simple CLI with CRUD commands.

### CODE REVIEW

**Review by ksvirkou-hubspot — APPROVED** [review]

File: node/src/cli.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  .command(authCommand)
+  .command(getCommand)
+  .command(createCommand)
+  .command(updateCommand)
+  .command(deleteCommand)
+  .command(propertiesCommand)
+  .help()
+  .recommendCommands()
+  .demandCommand(1, '')
+  .strictCommands().argv;
```
Comment (tpeuneu-hubspot) [inline node/src/cli.js]: I would add here the output of the help command
image

File: README.md [hunk 2]
```diff
@@ -1,2 +1,37 @@
-# manage_crm_objects
-Sample application in PHP, Python, Ruby and JavaScript demonstrating HubSpot API to manage CRM Objects
+# HubSpot CRM Objects sample app
+
+This is a sample app for the HubSpot [client libraries](https://developers.hubspot.com/docs/api/overview). This sample app demonstrates how to authenticate your HubSpot app using OAuth 2.0, and make an API call to different CRM Objects API.
```
Comment (atanasiuk-hubspot) [inline README.md]: Can we remove unrelated to this app description?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,37 @@
...
+
+## Reference
+
+- [Working with OAuth](https://developers.hubspot.com/docs/api/working-with-oauth)
```
Comment (atanasiuk-hubspot) [inline README.md]: Also unrelated references

File: README.md [hunk 4]
```diff
@@ -1,2 +1,37 @@
...
+cd node # there's a README in this folder with instructions
+npm install
+node src/cli.js
+```
+
+## Supported languages
+
+* [JavaScript (Node)](node/README.md)
+
+## Note on application scopes
```
Comment (atanasiuk-hubspot) [inline README.md]: And notes on applications scopes (app uses hapikey)

File: README.md (hunk 2 above)
Comment (asalauyeu-hubspot) [reply README.md]: Why it's unrelated?

File: README.md (hunk 3 above)
Comment (asalauyeu-hubspot) [reply README.md]: agree

File: README.md (hunk 2 above)
Comment (atanasiuk-hubspot) [reply README.md]: "This sample app demonstrates how to authenticate your HubSpot app using OAuth 2.0" - it doesn't, coz uses hapikey instead of oauth

Comment (asalauyeu-hubspot) [reply README.md]: oh, you mean phrase about oauth, got it

**Review by atanasiuk-hubspot — APPROVED** [review]

**Comment by tpeuneu-hubspot** [comment]
I suggest writing tests with the next PR

**Review by tpeuneu-hubspot — APPROVED** [review]

File: node/src/cli.js (hunk 1 above)
Comment (asalauyeu-hubspot) [reply node/src/cli.js]: I've already fixed that, through changing demand command logic, you can pull new version and take a look

**Pull request merged**